sympy/plotting/experimental_lambdify.py

```diff
@@ -239,6 +240,9 @@ def __call__(self, args):
 
 
 def experimental_lambdify(*args, **kwargs):
+    if kwargs.get('use_python_math', None) is True:
```

Comment:
Better to have `False` as default value?

Reply:
I dont think because if it is not initialised yet then it should be ```None``` , from what I had learned.
@smichr What do you think?

Reply:
I'm thinking from a "semantical" point of view, if `use_python_math` is not set, it is by default `False`. Still, your approach will work correctly as `True` != `None`.

Reply:
Often we use None when it isn't specified. This would mean, "do the default." True or False are used to indicate in the affirmative when it should or should not be done, respectively.